homeassistant/components/modbus/validators.py

```diff
@@ -280,6 +299,13 @@ def check_hvac_target_temp_registers(config: dict) -> dict:
         wrn = f"{CONF_FAN_MODE_REGISTER} overlaps CONF_TARGET_TEMP register(s). {CONF_FAN_MODE_REGISTER} is not loaded!"
         _LOGGER.warning(wrn)
         del config[CONF_FAN_MODE_REGISTER]
+    if (
```

Comment:
Are you testing that a list only contains 1 entry.

Reply:
This piece of code is not related to this PR, anyway the list cannot have more than 1 entry because it is checked at line 313 `def register_int_list_validator(value: Any) -> Any:`  and a proper warning is raised.